cmd/restic/global.go

```diff
@@ -105,6 +106,7 @@ func init() {
 	f.BoolVar(&globalOptions.NoCache, "no-cache", false, "do not use a local cache")
 	f.StringSliceVar(&globalOptions.CACerts, "cacert", nil, "`file` to load root certificates from (default: use system certificates)")
 	f.StringVar(&globalOptions.TLSClientCert, "tls-client-cert", "", "path to a file containing PEM encoded TLS client certificate and private key")
+	f.BoolVar(&globalOptions.InsecureSkipTLSVerify, "insecure-skip-tls-verify", false, "skip TLS certificate verification when connecting to the repo (insecure)")
```

Comment:
Please rename the flag to `--insecure-tls`. Although technically correct, the current name is rather hard to parse.